CodeSnippetsReflection.OpenAPI/LanguageGenerators/PhpGenerator.cs

```diff
@@ -106,9 +106,51 @@ public string GenerateCodeSnippet(SnippetModel snippetModel)
             WriteObjectProperty(RequestBodyVarName, payloadSb, codeGraph.Body, indentManager);
         }
         WriteRequestExecutionPath(codeGraph, payloadSb, indentManager);
+        var importStatements = GetImportStatements(snippetModel);
+        var phpTagIndex = payloadSb.ToString().IndexOf("<?php");
+        if (phpTagIndex >= 0)
+        {
+            var insertPosition = phpTagIndex + "<?php".Length + Environment.NewLine.Length;
+            payloadSb.Insert(insertPosition, string.Join(Environment.NewLine, importStatements) + Environment.NewLine);
+        }
+        else
+        {
+            payloadSb.Insert(0, string.Join(Environment.NewLine, importStatements) + Environment.NewLine);
+        }
         return payloadSb.ToString().Trim();
     }
+    private static HashSet<string> GetImportStatements(SnippetModel snippetModel)
+        {
+            const string modelImportPrefix = "use Microsoft\\Graph\\Generated\\Models";
+            const string requestBuilderImportPrefix = "use Microsoft\\Graph\\Generated";
+
+            var snippetImports = new HashSet<string>();
 
+            snippetImports.Add("use Microsoft\\Graph\\Graph;");
+
+            var importsGenerator = new ImportsGenerator();
+            var imports = importsGenerator.GenerateImportTemplates(snippetModel);
+            foreach (var import in imports)
+            {
+                switch (import.Kind)
+                {
+                    case ImportKind.Model:
+                        snippetImports.Add($"{modelImportPrefix};");
+                        // check if model has a nested namespace and append it to the import statement
+                        break;
+                    case ImportKind.Path:
+                        if (import.Path != null && import.RequestBuilderName != null)
```

Comment:
```suggestion
                        if (!string.IsNullOrEmpty(import.Path) && !string.IsNullOrEmpty(import.RequestBuilderName))
```
We are checking a string.